themes/finna2/templates/RecordDriver/SolrLido/core.phtml

```diff
@@ -97,21 +98,53 @@
           <?=$this->record($this->driver)->renderTemplate('versions-link.phtml')?>
         <?php endif; ?>
       </div>
-
       <?php if ($documents = $this->driver->getDocuments()): ?>
         <div class="record-links">
           <div class="recordURLs local-available-online-record">
             <ul class="truncate-field outer" data-rows="4">
               <?php foreach ($documents as $i => $document): ?>
                 <?php
                   $description = $document['description'] ?: $this->translate('Download the file');
+                  $rights = $this->driver->getAccessRestrictionsType($language);
+                  $rightsDescription = $this->driver->getAccessRestrictions($language);
+                  $hasLink = isset($rights['link']) && !empty($rights['link']);
                 ?>
                 <li class="fulltextField">
-                  <a href="<?=$this->escapeHtmlAttr($this->url('record-downloadfile', [], ['query' => ['id' => $this->driver->getUniqueID(), 'index' => $i, 'format' => $document['format'], 'type' => 'document']]))?>" target="_blank" download>
-                    <i class="fa fa-download" aria-hidden="true"></i>
-                    <?=$this->escapeHtml($description)?>
-                  </a>
-                  (<?=$this->escapeHtml($document['format'])?>)
+                  <div class="link-wrapper">  
+                    <a href="<?=$this->escapeHtmlAttr($this->url('record-downloadfile', [], ['query' => ['id' => $this->driver->getUniqueID(), 'index' => $i, 'format' => $document['format'], 'type' => 'document']]))?>" target="_blank" download>
+                      <i class="fa fa-download" aria-hidden="true"></i>
+                      <?=$this->escapeHtml($description)?>
+                    </a>
+                    <div class="copyright-wrapper">
+                      <span>
+                        (<?=$this->escapeHtml($document['format'])?><?php if ($rights) { ?>:<?php } ?>
+                      </span>
+                      <span class="copyright-icons">
+                        <?=$this->partial('Helpers/copyright-icons.phtml', ['copyright' => $rights['copyright']])?>
+                      </span>
+                      <?php if ($hasLink): ?>
+                        <a class="copyright-meaning" href="<?=$this->escapeHtmlAttr($rights['link'])?>" target="_blank"><?=$this->escapeHtmlAttr($rights['copyright']);?></a>
+                      <?php else: ?>
+                        <span>
+                          <?=$this->escapeHtmlAttr($rights['copyright']);?>
+                        </span>
+                      <?php endif; ?>
+                      <button class="show-info cc-info more-link">
+                        <?=$this->transEsc('Additional Information')?> <i class="fa fa-arrow-down"></i>)
```

Comment:
Käytetään uudessa koodissa icon helperiä, esim. `<?=$this->icon('truncate-more')?>` ja `<?=$this->icon('truncate-less')?>`